.release/buildspec_e2e.yml

```diff
@@ -21,6 +21,8 @@ batch:
           TEST_SUITE: addons
           APP_REGION: us-west-2
           APP_ACCOUNT: e2e/account/addons
+          TEST_REGION: us-west-2
+          TEST_ACCOUNT: e2e/account/addons
```

Comment:
Can we make it optional and by default to be the same as app? Same for the region.

Reply:
I sort of wanted these information to be as obvious and plain-text-ed as possible. Like I don't want people to have to look at `phases.build` in order to understand what account / region is being used. 

Are you concerned about the number of lines in the codebuild?